### PR TITLE
refactor(setup): fix for docker-compose location issues

### PR DESCRIPTION
**Pull request by voryzen**
FIX:
When pip installs docker-compose, it gets installed in /usr/local/bin, not /usr/bin, as per the service files.
It appears to happen on debian and on centos.

### CODE REVIEW

File: roles/vivumlab_setup/tasks/debinstall.yml [hunk 1]
```diff
@@ -104,6 +104,19 @@
     executable: pip3
   become: yes
 
+- name: Check for Docker-compose
+  stat:
+    path: /usr/bin/docker-compose
```
Comment (codefriar) [inline roles/vivumlab_setup/tasks/debinstall.yml:109]: why not just tell Ansible to always use $(which docker-compose) ?

Comment (voryzen) [reply roles/vivumlab_setup/tasks/debinstall.yml:109]: Multiple installs of docker-compose.

Sometimes docker-compose is preinstalled on some OS's or installed via it's package management system; 

Vivum uses pip to install docker-compose, and its always up to date..

Any ideas here?

**Comment by denis-ev** [comment]
There is still a review open, I will wait with merging.
@voryzen

**Comment by voryzen** [comment]
For the sake of having Debian users able to use Vivum, we can discuss the new idea, after it has been tested

**Comment by voryzen** [comment]
@denis-ev let's merge, mate

**Review by denis-ev — APPROVED** [review]

**Pull request merged**